services/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -62,22 +62,23 @@ object UserService {
         request: PasswordUpdateRequest,
```

Comment:
Is this just a refactoring or are these changes connected to the current task? Anyway, i would appreciate a small explanation on what is the main reason for these changes.

Reply:
I just a bit rewrote the implementation of this method since  the previous version was a little confusing.